mautrix_telegram/matrix.py

```diff
@@ -69,16 +71,71 @@ async def handle_puppet_group_invite(
         evt: StateEvent,
         members: list[UserID],
     ) -> None:
-        if self.az.bot_mxid not in members:
+        double_puppet = await pu.Puppet.get_by_custom_mxid(invited_by.mxid)
```

Comment:
This should get the user instance and make sure it's logged in, not the double puppet